fedbiomed/common/constants.py

```diff
@@ -53,8 +59,8 @@
 # major version, else the minor version. Micro versions are supported but their use is currently discouraged.
 
 __version__ = FBM_Component_Version('5.0.0')  # Fed-BioMed software version
-__researcher_config_version__ = FBM_Component_Version('2')  # researcher config file version
-__node_config_version__ = FBM_Component_Version('2')  # node config file version
+__researcher_config_version__ = FBM_Component_Version('3')  # researcher config file version
```

Comment:
I think it is normal to break configuration version (or any version) in develop because it is a development branch. If you add version 2 and then 3 in the develop the master will jump from 1 to 3 directly.

Reply:
... but on the other hand if we don't do that we'll have to send yet-another warning message when merging this branch because it will break the configurations of most of us who already are using the develop branch ... as it did for me and I spent quite a while trying to figure why I got an ugly stack trace (instead of a clean "version changed" message). 

I can revert to v2, but I'll let you communicate to other people about that when we merge the branch :-)